crates/common/task_executor/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "task_executor"
+authors.workspace = true
+edition.workspace = true
+keywords.workspace = true
+license.workspace = true
+readme.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+[dependencies]
+tokio = { workspace = true}
+futures = {workspace = true}
+async-channel = {workspace = true}
```

```suggestion
tokio = { workspace = true }
futures = {workspace = true }
async-channel = {workspace = true }
```

missing a new line at the bottom and depends aren't sorted, also please make spacing consistent